index.html

```diff
@@ -18,20 +18,160 @@
         </nav>   
     </header>
     <section>
-            <h1 class="perso">My name is Dany<br> I am glad to meet you!</h1>
-            <p class="pro"> I’m a software developer! I can help you build a product ,
+            <h1 class=Perso">My name is Amiral<br> I am glad to meet you!</h1>
+            <p class="bio"> I’m a software developer! I can help you build a product ,
             feature or website Look through some of my work and experience!
             If you like what you see and have a project you need coded,
             don’t hestiate to contact me.</p>
 
             <h2 class="connect">LET'S CONNECT</h2>
-            <ul class="icon">
+            <ul class="icon-bio">
                 <li><a href="www.twitter.com/amiralda"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
                 <li><a href="https://www.linkedin.com/danyaugustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
                 <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
                 <li><a href="https://www.medium.com/amiralda/"><img src="img/medium.svg" alt="medium-icon"></a></li>
                 <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
             </ul>
     </section>
+
+<section class="Parent">
+    <article class="box">
+        <img class="image" src="img/tonic.png" alt="tonic">
+        <div class="box-info">
+            <h2 class="title">Tonic </h2>
+            <div class="description">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="img/dot.png" alt="dot">
+                <a href="#" class="tech">BACK END DEV</a>
+                <img class="dot" src="img/dot.png" alt="dot">
+                <a href="#" class="year">2015</a>
+            </div>
```

Comment:
- [ ] You did a fantastic job adding the `work` section to your project :+1:, but to match as much as possible to the Figma template you should make some changes listed below:
- I've noticed that your `works` section is overlapping with the background of your home page, kindly fix it to match the template.
- I think it would be better to change the background of the works section, it means instead of using #fff your should use #e5e5e5
- Kindly center the image to the middle of each card to match the template

**Your design**

![Screenshot from 2023-02-25 07-08-35](https://user-images.githubusercontent.com/105117832/221356622-788c33cb-fe73-4f6d-959b-07ebad70b8df.png)

![Screenshot from 2023-02-25 07-11-45](https://user-images.githubusercontent.com/105117832/221356627-09f0988f-8007-4dd4-b658-4f3ba6267dce.png)

**Figma**

![Screenshot from 2023-02-25 07-12-42](https://user-images.githubusercontent.com/105117832/221356637-3469341d-e2e0-4e85-bd3c-fc13748679b1.png)

![Screenshot from 2023-02-25 07-25-20](https://user-images.githubusercontent.com/105117832/221356686-0e2079f4-9278-4e9f-acc1-b913ea8304fe.png)

Reply:
For me this work well, in attachment a screenshot on how it is on my computer.
![review1](https://user-images.githubusercontent.com/28764819/221380402-275a7eb3-b083-4c34-9d72-2fc12ce21e86.JPG)

Reply:
- [ Optional ] Hello @amiralda the above Screen-shot you provide is at screen size of 493px but Minimum mobile screen size should be 375px, at the screen size of 375px it's overlapping kindly adjust it.  
Your Work section at 493px
![Capture](https://user-images.githubusercontent.com/93598189/221381883-080be9f0-e5b2-448b-a2c7-ca50ba3daaa6.PNG)

Your Work section (below 493px, for Example at 375px screen size)

![Capture](https://user-images.githubusercontent.com/93598189/221381967-8a7d518c-c5a7-42c8-bfc7-eaf99633e94b.PNG)